src/main/java/org/folio/dataexp/service/logs/ErrorLogService.java

```diff
@@ -90,6 +93,50 @@ public ErrorLog saveGeneralErrorWithMessageValues(String errorMessageCode, List<
     return save(errorLog);
   }
 
+  public void saveCommonExportFailsErrors(CommonExportFails commonExportFails, int totalErrors, UUID jobExecutionId) {
```

Comment:
It looks like inconsistency in methods signatures. Methods saveGeneralErrorWithMessageValues, saveWithAffectedRecord, etc. return ErrorLog type, but saveCommonExportFailsErrors has void type. Can all these methods be aligned?